### DISCUSSION TITLE
What is your experience of using this tool?

### DISCUSSION THREAD

**Discussion by dedezagaha** [raingggg/coctohug#3] [Q&A] [answered]
Compared with the official chia farming, do you have a different feeling by using this tool?

**Comment by jamisshow** [comment, marked as answer]
If you plot size is bigger than 200T, you will earn fork coins every hour! This is super cool, and gives me more expectation. For the official chia client, I had to wait months and always doubt "is my configuration good enough?"

This is my earned coins screenshot

**Reply by jamisshow** [reply]
![earned_coins](https://user-images.githubusercontent.com/95666045/144957775-c0a2634c-bdc4-4e8f-8aba-477894a767de.png)

**Comment by raingggg** [comment]
Glad to see you guys here! We are adding more forks.
